package.json

```diff
@@ -63,10 +63,12 @@
     "aegir": "^10.0.0",
     "chai": "^3.5.0",
     "eslint-plugin-react": "^6.9.0",
+    "go-ipfs-dep": "^0.4.5",
     "gulp": "^3.9.1",
     "hapi": "^16.1.0",
-    "interface-ipfs-core": "~0.24.1",
+    "interface-ipfs-core": "github:ipfs/interface-ipfs-core#fix\/pubsub-tests",
```

Comment:
can I get a PR on interface-ipfs-core and a note here that says to use that one? Remember how this 'pointing to branches and commits' has bitten us in the past, let's stop using them as we talked in January.